### ISSUE TITLE
File Truncation?

### ISSUE THREAD

**Issue by No9** [dominictarr/tacodb#1]
Got a fork of taco-demo and included rickshaw which has a dependency on d3. 
When `d3.v2.js` is sent to the server during `./build.sh 8000` 
The end of the file appears to be removed or isn't served properly afterwards. 
This leads to the following error.

```
Timestamp: 08/06/13 19:03:33
Error: SyntaxError: missing ) after argument list
Source File: http://localhost:8000/http/taco-demo/vendor/d3.v2.js
Line: 7024, Column: 60
Source Code:
le.linear(), d3_time_scaleUTCMethods, d3_time_scaleUTCFormat
```

Checkout https://github.com/No9/taco-demo and see 
https://github.com/No9/taco-demo/blob/master/static/index.html#L13

For steps to reproduce. There is a work around in there for now but will come back to this.

**Comment by dominictarr** [comment]
I think this is a problem with level-static seems like it's truncating the end of the file

```
~/c/taco-demo>curl localhost:8000/http/taco-demo/vendor/d3.v2.js > tmp
~/c/taco-demo>diff tmp static/vendor/d3.v2.js
7024c7024,7026
<     return d3_time_scale(d3.scale.linear(), d3_time_scaleUTCMethods, d3_time_scaleUTCFormat
\ No newline at end of file
---
>     return d3_time_scale(d3.scale.linear(), d3_time_scaleUTCMethods, d3_time_scaleUTCFormat);
>   };
> })();
```

looking into this...

**Comment by dominictarr** [comment]
okay, I figured this out.

the problem was that I was doing `res.setHeader('content-length', string.length)`
and `string.length !== new Buffer(string.length)` if there is special chars in there.

this is fixed in level-static@1.0.8
and in tacodb@1.0.9
